fuzzing/private/oss_fuzz/repository.bzl

```diff
@@ -14,19 +14,51 @@
 
 """Repository rule for configuring the OSS-Fuzz engine and instrumentation."""
 
+_UBSAN_STANDALONE_CXX_LIB_NAME = "libclang_rt.ubsan_standalone_cxx-x86_64.a"
+
 def _to_list_repr(elements):
     return ", ".join([repr(element) for element in elements])
 
+def _find_llvm_lib(repository_ctx, target_file):
+    result = repository_ctx.execute([
+        repository_ctx.which("bash"),
+        "-c",
+        """
+        set -euf -o pipefail
+        set -x
+        find "$({llvm_config} --libdir)" -name {target_file} | head -1
+        """.format(
+            llvm_config = "llvm-config",
+            target_file = target_file,
+        ),
+    ], quiet = False)
+    file_path = result.stdout.strip()
```

Comment:
Can result.stdout be None in any case, that can cause an exception.

Reply:
I don't think this can be None. According to [the docs](https://docs.bazel.build/versions/master/skylark/lib/exec_result.html), this is always set to the stdout contents of the program (i.e., an empty string if there's no output).

Reply:
sg, skip!